src/index.ts

```diff
@@ -22,22 +23,32 @@ const initApiCall = async (
   link?: string
 ) => {
   try {
+    const init_info = await Promise.all([
+      device_data(),
+      getPushToken(),
```

Comment:
Did not add `getPushToken` in the `device_data` function, because `device_data` is used at other places which does not require push token info